.kitchen.yml

```diff
@@ -27,6 +27,19 @@ verifier:
   name: terraform
 
 suites:
+  - name: "simple_instance"
```

Comment:
I'm not sure why this needs to be a test. Especially not sure why we have *both* examples/simple and test/fixtures/simple_instance. This seems overly complicated to be adding 16(!) files just for a single simple example.

Reply:
@morgante Please see https://github.com/terraform-google-modules/terraform-google-container-vm/pull/3#issuecomment-430618247